docs/CLI/index.md

```diff
@@ -10,37 +10,47 @@ Install `oras` using [Homebrew](https://brew.sh/):
 brew install oras
 ```
 
+> Note: Homebrew only has ORAS v0.16.0 available to be used so far. It will be updated when ORAS v1.0.0 is released in Mar 2023.
+
 ## Release artifacts
 
-Install from the latest [release artifacts](https://github.com/oras-project/oras/releases):
+Install ORAS from the latest [release artifacts](https://github.com/oras-project/oras/releases):
 
 ### Linux
 
+If you want to install ORAS on an AMD64-based Linux machine, run the following command:
+
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_linux_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_amd64.tar.gz
 mkdir -p oras-install/
-tar -zxf oras_0.16.0_*.tar.gz -C oras-install/
-mv oras-install/oras /usr/local/bin/
-rm -rf oras_0.16.0_*.tar.gz oras-install/
+tar -zxf oras_1.0.0-rc.1_*.tar.gz -C oras-install/
+sudo mv oras-install/oras /usr/local/bin/
+rm -rf oras_1.0.0-rc.1_*.tar.gz oras-install/
 ```
 
+> Note: If you want to install ORAS on an ARM64-based Linux machine, you can download it from `https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_arm64.tar.gz`.
+
 ### macOS
 
```

Comment:
Shouldn’t this be Mac and not arm64?

Reply:
This is the mac section and new ones are arm, although I have an amd one as well.

Reply:
I like the `ARCH=$(test $(uname -m) = arm64 && echo arm64 || echo amd64)` which would work for mac or Linux, but not a big deal.

Reply:
For arm64, the `uname -m` result might be `aarch64`

Reply:
> I like the `ARCH=$(test $(uname -m) = arm64 && echo arm64 || echo amd64)` which would work for mac or Linux, but not a big deal.

I think we can keep it hard coded for now. We probably need a shell script to make it one line for all platforms and OSs.

Reply:
@shizhMSFT @TerryHowe  @qweeah  Developing a shell script for all platforms/OS sounds much better. I expect that install ORAS via the following sample commands:

```
# Download and install the latest version
curl -L https://oras.land/intalloras sh -

# Download and install a specific version with a target architecture
curl -L https://oras.land/intalloras | ORAS_VERSION=v1.0.0 ARCH=x86_64 sh -
```

How about I create a new issue to track this feature request?

Reply:
Those style installers are convenient, but always make me a bit uneasy because it isn't obvious what it is going to do.

Reply:
It follows the get.docker.com model. I think it’s ok but means we have to maintain multiple channels of installation. Not sure if it’s worth the effort rather than just generating a good doc that you can copy paste.

Reply:
If I am a new  oras user, an installation script sounds way more friendly than a well-written doc. Yes I am kind of user that don't read doc until get blocked.

Reply:
Thank @TerryHowe @sajayantony @qweeah for your suggestions. It sounds like we all agree that this installation script is nice to have. I will create an issue to track it.